apps/web/app/(org)/dashboard/settings/organization/components/CustomDomainDialog/CustomDomainDialog.tsx

```
@@ -0,0 +1,436 @@
+        errors: {}
+      };
+
+    case 'PREV_STEP':
+      return {
+        ...state,
+        currentIndex: Math.max(state.currentIndex - 1, 0),
+        errors: {}
+      };
+
+    case 'GO_TO_STEP':
+      const targetIndex = action.payload;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)